packages/puppeteer-extra-plugin-recaptcha/src/index.ts

```diff
@@ -72,6 +72,11 @@ export class PuppeteerExtraPluginRecaptcha extends PuppeteerExtraPlugin {
       scriptSource = HcaptchaContentScript.toString()
       scriptName = 'HcaptchaContentScript'
     }
+    // Some bundlers transform classes to anonymous classes that are assigned to
+    // vars (e.g. esbuild). In such cases, `unexpected token '{'` errors are thrown
+    // once the script is executed. Let's bring class name back to script in such
+    // cases!
+    scriptSource = scriptSource.replace(/class \{/, `class ${scriptName} {`)
```

Comment:
Hey :wave:

Mind co-authoring these?

Reply:
I think that simply re-comitting (remove -> add) just those lines with co-author commit message body will do. I'll close my PR once its done. Your description describes the problem better.